mongo/mongo.go

```diff
@@ -523,7 +543,21 @@ func ensureServer(args EnsureServerParams, mongoKernelTweaks map[string]string)
 			if err != nil {
 				return zeroVersion, errors.Trace(err)
 			}
-
+			if featureflag.Enabled(feature.MongoDbSnap) {
```

Comment:
Probably time to introduce a new function for the "if exists" branch given all the indenting etc.
But this is the wrong place for all this.
Setting up mongo should be part of the ServiceActions.Install(), ie implemented in service/snap.go